app/forms/idv/state_id_form.rb

```diff
@@ -28,7 +29,7 @@ def submit(params)
       cleaned_errors.delete(:identity_doc_address2, :nontransliterable_field)
 
       FormResponse.new(
-        success: valid?,
+        success: validation_success,
```

Comment:
Should we mark success as false by check cleaned_errors if any errors left?

Reply:
Looks like not.

Reply:
No, `cleaned_errors` is only here to prevent error messages that may contain PII from being logged (previously the errors were passed in directly).